exercise/templates/exercise/_exercise_info.html

```diff
@@ -50,8 +50,19 @@ <h4 class="panel-title">{% trans "Exercise info" %}</h4>
             <dd class="exercise-info-required-points">{{ exercise.points_to_pass }}</dd>
             {% endif %}
 
+
+
             <dt>{% trans "Deadline" %}</dt>
             <dd class="exercise-info-deadline">{{ exercise.course_module.closing_time }}</dd>
+            {% if exercise|deviation_deadline:profile %}
```

Comment:
There are three template tags here that each query the database for the same deviation. Only one database query should be used for the same thing.

Reply:
The `with` template tag should be useful for saving the deviation into a variable so that you can reuse it: 
https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#with